doc/architecture/decisions/0023-use-docker-hub-in-deployments.md

```diff
@@ -0,0 +1,33 @@
+# 23. use-docker-hub-in-deployments
+
+Date: 2020-10-01
+
+## Status
+
+Accepted
+
+## Context
+
+Our CI/CD pipeline uses containers (Docker) as does our hosting platform (GOVUK
+PaaS), we need a way to store built images from our pipeline so our hosting
+platform can access and deploy them.
+
+Docker hub is one solution offered by the maker of Docker itself.
+
+## Decision
+
+Use [Docker hub](https://hub.docker.com/) to store built deployment container
+images to facilitate continuous delivery.
+
+Host the built container images on the dxw Docker hub account.
+
+## Consequences
+
+Built container images will be stored on the dxw Docker hub account. This splits 
+the ownership of the dependencies the service is built on. This can be rectified
```

Comment:
`This splits 
the ownership of the dependencies the service is built on.`

 I'd change it to `This splits 
the ownership of the dependencies the service is built on between BEIS and dxw` as I think that's clearer.